### ISSUE TITLE
Medium trust 4.0 failure

### ISSUE THREAD

**Issue by mattsummers** [jamietre/CsQuery#108]
CQ document = CQ.Create(html);

---

Inheritance security rules violated while overriding member: 'CsQuery.Implementation.CombinedTextReader.CreateObjRef(System.Type)'. Security accessibility of the overriding method must match the security accessibility of the method being overriden.

Can this be resolved?

Thank you.
